README.md

```diff
@@ -1,168 +1,31 @@
 # Geoclimate
 
-Geoclimate chain is a collection of spatial processes to produce vector maps of cities. First, algorithms are used to extract and transform [OpenStreetMap](https://www.openstreetmap.org) (OSM) data to a set of GIS layers (GeoClimate abstract model). The GIS layers are then processed  to compute urban indicators at three geographic scales : RSU, block and building ([Bocher et al, 2018](http://dx.doi.org/10.1016/j.uclim.2018.01.008)). These indicators are used to feed the [TEB](http://www.umr-cnrm.fr/spip.php?article199) model, classify the urban tissues and build the *Local Climate Zones* ([LCZ](http://www.wudapt.org/lcz/)).
```

Comment:
This readme must be replaced by https://github.com/orbisgis/geoclimate/blob/master/docs/README.md

Reply:
You will have two times the same file ?

Reply:
The root README.md file has been removed. Github will automatically recognize the README.md in /docs/